### PR TITLE
chore(ci): Lower soak noise threshold

### PR DESCRIPTION
**Pull request by jszwedko**
Following some discussion about the summary frequently missing real
changes.

Anecdotally we've seen noise around +/- 2%. We'll try 5% for now.

Also adds this note to the summary table because there was some
confusion about why some changes didn't appear there since they seemed
to meet the criteria (the collapsed explanation section contains the
relevant details).

Signed-off-by: Jesse Szwedko <jesse@szwedko.me>

### CODE REVIEW

**Review by spencergilbert — APPROVED** [review]

**Review by fuchsnj — APPROVED** [review]

File: soaks/bin/analyze_experiment [hunk 1]
```diff
@@ -12,7 +12,7 @@ parser.add_argument('--baseline-sha', type=str, help='the sha of the baseline ex
 parser.add_argument('--capture-dir', type=str, help='the directory to search for capture files')
 parser.add_argument('--comparison-sha', type=str, help='the sha of the comparison experiment')
 parser.add_argument('--erratic-soaks', type=str, default='', help='a comma separated list of known-erratic experiments, NOT TO BE USED LIGHTLY')
-parser.add_argument('--mean-drift-percentage', type=float, default=8.87, help='the percentage of mean drift we allow in an experiment, expressed as a value from 0 to 100, default 9th percentile')
+parser.add_argument('--mean-drift-percentage', type=float, default=5, help='the percentage of mean drift we allow in an experiment, expressed as a value from 0 to 100, default 5')
```
Comment (blt) [inline soaks/bin/analyze_experiment:15]: This also needs to be updated in `detect_regressions`, or passed in the workflows.

**Review by blt — CHANGES_REQUESTED** [review]

File: soaks/bin/analyze_experiment (hunk 1 above)
Comment (jszwedko) [reply soaks/bin/analyze_experiment:15]: Aha, thank you for catching that.

Comment (jszwedko) [reply soaks/bin/analyze_experiment:15]: 765cd7b6a

**Review by blt — APPROVED** [review]

**Comment by blt** [comment]
> No interesting changes in throughput with confidence ≥ 90.00% and absolute Δ mean >= ±5%:

So nice.

**Pull request merged**